Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -131,11 +131,22 @@ public final class PledgeCTAContainerViewViewModel: PledgeCTAContainerViewViewMo
 // MARK: - Functions
 
 private func pledgeCTA(project: Project, backing: Backing?) -> PledgeStateCTAType {
+  let uuid = (AppEnvironment.current.device.identifierForVendor
+    ?? UUID()).uuidString // TODO: move to environment
```

Comment:
Move to environment in order to mock a UUID string?

If you just want to test whether a UUID string was returned you can do:
```swift
XCTAssertNotNil(UUID(uuidString: uuidString))
```
It'll be `nil` if it's not a valid UUID string.

Reply:
Hmm maybe a helper function would be better here. I'll add one.